dev-scripts/format-all.sh

```diff
@@ -26,6 +26,7 @@ if ! which "$GOPATH"/addlicense; then
     exit 1
 fi
 
+"$GOPATH"/addlicense -c "Google LLC" -l apache "$GIT_ROOT"
```

Comment:
Moving this earlier so that it still adds the license even if a later format command fails (like if I don't have an AAOS Unbundled checkout handy)